### ISSUE TITLE
[Video - .wmv file type] is rendered as Audio file type on server side

### ISSUE THREAD

**Issue by kjnanda** [rtMediaWP/transcoder#80]
**Steps:**

1. Upload video file with `.wmv` supported format from `media.php file` (Or from any other location)
2. Wait for the file to be transcoded.
3. Observe on the server side where it renders this file type as `audio` file type and hence it does not show up the status as `File is transcoded` under the status column once transcoding is done.

for `point#3` the separate issue has been created. Refer - https://github.com/rtMediaWP/transcoder/issues/79

-----

```
Test Environment : 

1. WP - V5.0.3
2. Transcoder plugin V1.2.2
3. Test Url - https://dev.transcoder.dev2.rt.gw
```

**Comment by kjnanda** [comment]
**Please make a note that - Above file is not playing in the front end. And simply shows a link. Clicking upon that link, it starts downloading the file but not playing.**
